com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -489,24 +518,40 @@ private void RegisterPickupAction()
             interactable.OnClicked.AddListener(StartPlacement);
         }
 
+        /// <summary>
+        /// Registers the event which performs placement.
+        /// </summary>
+        private void RegisterPlacementEvent()
+        {
+            if (interactable == null)
+            {
+                Debug.Log("Failed to register the placement event. There is no StatefulInteractable set.");
+                return;
+            }
+            interactable.OnClicked.AddListener(StopPlacement);
+        }
+
         /// <summary>
         /// Unregisters the input action which performs placement.
         /// </summary>
         private void UnregisterPlacementAction()
         {
-            InputAction placementAction = GetInputActionFromReference(placementActionReference);
-            if (placementAction == null)
+            foreach (InputAction placementAction in placementActions)
```

Comment:
looks like `placementActions` should be `lastActions`

Reply:
good catch